onnx/defs/nn/defs.cc

```diff
@@ -2529,8 +2546,7 @@ ONNX_OPERATOR_SET_SCHEMA(
           // Output Y has same shape as X
           // Outputs Mean and InvStdDev have shape: [d[0], ..., d[axis-1], 1, ..., 1]
           FunctionBuilder builder(functionProto);
-          builder.AddOpset("", 16)
-              .Const("FloatEpsilon", ToTensor<float>(epsilon))
+          builder.Const("FloatEpsilon", ToTensor<float>(epsilon))
```

Comment:
Why change this? Seems unrelated to the main PR. Anyway: we can decide on this after the decision about opsets in function-definitions.

Reply:
I discussed with the original author, this AddOpset("", 16) was added due to a typo. It was detected by the change that use the function's opset version if specified in test case generation.